modules/blog/blog-home/blog-home.module.scss

```diff
@@ -0,0 +1,106 @@
+@import '../../../styles/variables';
+@import '../../../styles/mixins';
+
+.container {
+  align-items: center;
+  column-gap: 50px;
+  display: flex;
+  justify-content: center;
+  @include breakpoint(small) {
+    flex-direction: column-reverse;
+    padding-top: 0;
+    margin-bottom: 3.75rem;
+  }
+}
+
+.containerRowReverse {
+  flex-direction: row-reverse;
+
+  @include breakpoint(small) {
+    flex-direction: column-reverse;
+  }
+}
+
+.info,
+.image {
+  width: 50%;
+
+  @include breakpoint(small) {
+    width: 100%;
+  }
+}
+
+.image {
+  display: flex;
+  justify-content: center;
+}
+
+.info {
+  .title {
+    @include h1;
+    line-height: 5rem;
+    margin-top: 0.75rem;
+  }
+
+  .description {
+    @include h3;
+  }
+}
+
+.isHovered {
+  cursor: pointer;
+  color: yellow;
+  text-decoration-line: underline;
+}
+
+.infoMedium {
+  .title {
+    @include h2;
+    line-height: 2.375rem;
+    margin-top: 1rem;
+  }
+
+  .description {
+    @include h4;
+  }
+}
+
+.imageMedium {
+  margin: 7rem 0;
+
+  @include breakpoint(small) {
+    margin: 0;
+  }
+}
+
+.author,
+.date,
+.category {
+  font-size: 0.875rem;
+}
+
+.category {
+  margin-bottom: 0;
+}
```

Comment:
Looks like this should have a color of `Color/Secondary/Grey/grey-500`.

Reply:
ah yeah good spot, updated now